docs/source/spec/xml.rst

```diff
@@ -221,8 +221,9 @@ following document:
     ``-``. Values for an ``xmlName`` adhere to the following ABNF.
 
 .. productionlist:: smithy
-    xml_name       :(ALPHA / "_")
+    name           :(ALPHA / "_")
```

Comment:
Do we already use the name production anywhere else? I think this might break how we link to them across "smithy" production lists. Maybe `xml_name` and `xml_identifier`?